src/CodeGeneration/ApiGenerator/Views/ElasticLowLevelClient.Generated.cshtml

```diff
@@ -16,8 +16,7 @@ using System.Threading.Tasks;
 using Elasticsearch.Net;
 using static Elasticsearch.Net.HttpMethod;
 
-//This file is automatically generated from https://github.com/elastic/elasticsearch/tree/@Model.Commit/rest-api-spec
-//Generated from commit @Model.Commit
+//This file is automatically generated from https://github.com/elastic/elasticsearch/tree/master/rest-api-spec
```

Comment:
Should this include `@Model.Commit`?

Reply:
No since I quite often end up copying the files over manually (especially from x-pack) this goes out of sync.  I want to set up a public repos with the rest spec that is sync'ed automatically to publish them more formally.